File: C/sumOfDigits.c
#include <stdio.h>

int main(void)
{
    int t, digit, sum=0;
    scanf("%d", &t);
    
    while(t--)
    {
        scanf("%d", &digit);
        while(digit!=0)
        {
            sum+=digit%10;
            digit/=10;
        }
        printf("%d\n", sum);
        sum=0;
    }
    return 0;
}

File: Java/SumofDigits.java
import java.util.*;  
class SumOfDigits  
{  
	public static void main(String args[])  
	{  
		int n, digit, sum = 0;  
		Scanner sc = new Scanner(System.in);   
		n= sc.nextInt();  
		while(n>0)  
		{  
			digit = n%10;  
			sum = sum + digit;  
			n= n/10;  
		}    
		System.out.println(sum);  
	}  
}  
